tests/libraries/math/test_safeMathUint256.py

```diff
@@ -4,14 +4,25 @@
 from ethereum.tools.tester import TransactionFailed
 from pytest import fixture, mark, lazy_fixture, raises
 
+@fixture(scope='session')
+def testerSnapshot(sessionFixture):
+    sessionFixture.uploadAndAddToController('solidity_test_helpers/SafeMathUint256Tester.sol')
```

Comment:
This needs to do `sessionFixture.resetSnapshot()` first, otherwise it could end up working with an already used fixture.